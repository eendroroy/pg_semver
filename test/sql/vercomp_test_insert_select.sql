LOAD 'vercomp';

CREATE TABLE versions(version VERSION);

INSERT INTO versions VALUES ('1.0.0'), ('0.0.0'), ('1.1.0'), ('1.2.0'), ('1.23.0'), ('1.23.9'), ('2.9.0'), ('2.0.12'),
                            ('2.0.0-alpha'), ('3.0.1'), ('3.0.2'), ('3.0.4-beta2'), ('3.0.4-alpha'), ('3.0.4-rc1'),
                            ('1.5.0'), ('1.05.0'), ('1.005.0');

SELECT * FROM versions;

DROP TABLE IF EXISTS versions;
