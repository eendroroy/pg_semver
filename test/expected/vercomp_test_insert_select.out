LOAD 'vercomp';
CREATE TABLE versions(version VERSION);
INSERT INTO versions VALUES ('0.0.0'), ('1.0.0'), ('2.0.0-alpha'), ('2.0.0-beta0'), ('2.0.0-beta1'), ('2.0.0-rc1'), ('3.0.0');
SELECT * FROM versions;
   version   
-------------
 0.0.0
 1.0.0
 2.0.0-alpha
 2.0.0-beta0
 2.0.0-beta1
 2.0.0-rc1
 3.0.0
(7 rows)

DROP TABLE IF EXISTS versions;
